CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Master (Unreleased)
 
+## 2.23.0 (2023-05-28)
+
+- Update the message output of `RSpec/ExpectActual` to include the word 'value'. ([@corydiamand])
```

Comment:
Ah, I see. `### 2.23.0 (2023-05-28)` is what you added. Could you change it as follows?

```suggestion
## Master (Unreleased)

- Update the message output of `RSpec/ExpectActual` to include the word 'value'. ([@corydiamand])

```